### PR TITLE
task 6

### PR DESCRIPTION
**Pull request by Viccart**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -120,3 +120,64 @@ describe("GET /api/articles", () => {
...
+        expect(comments).toBeInstanceOf(Array);
+        expect(comments).toHaveLength(1);
+        expect.objectContaining({
+          comment_id: expect.any(null),
+          votes: expect.any(null),
+          created_at: expect.any(null),
+          author: expect.any(null),
+          body: expect.any(null),
+          article_id: expect.any(Number),
+        });
```
Comment (HeatherEver) [inline __tests__/app.test.js:162]: this test should just really return an empty array - have a look at your join further down!

File: models/articles.models.js [hunk 2]
```diff
@@ -28,3 +28,25 @@ exports.selectArticles = () => {
...
+
+exports.selectArticleCommentsById = (articleID) => {
+  return db
+    .query(
+      `SELECT comments.comment_id, comments.votes, comments.created_at, comments.author, comments.body, articles.article_id
+        FROM comments
+        RIGHT JOIN articles
+        ON articles.article_id = comments.article_id
+        WHERE articles.article_id = $1
+        ORDER BY created_at DESC;`,
```
Comment (HeatherEver) [inline models/articles.models.js:40]: so this is a good idea but if there are no comments we're still getting information back that we don't require - is there another way we could check the the article_id exists and if it does exist but the rowCount is 0 send back empty array...

**Review by HeatherEver — APPROVED** [review]
Lovely work Hannah, just your SQL query needs a bit of altering - i have left some hints!

File: models/articles.models.js (hunk 2 above)
Comment (Viccart) [reply models/articles.models.js:40]: thanks Heather, I've made the changes, do I have to hold fire and submit another pr to get it authorized?

**Review by pippaamy — APPROVED** [review]
Great work Hannah, go ahead and merge!

**Pull request merged**